dotcom-rendering/src/web/components/OnwardsData.tsx

```diff
@@ -35,18 +56,51 @@ export const OnwardsData = ({
 		return trails.slice(0, trailLimit).map(decideTrail);
 	};
 
-	if (data && data.trails) {
+	useEffect(() => {
+		if (data) {
+			const pendingElements = document?.querySelectorAll<HTMLElement>(
+				'.onwards > .pending',
+			);
+			pendingElements?.forEach((element) => {
+				element.classList.add('reveal');
+				element.classList.remove('pending');
+			});
+		}
+	});
+
+	if (error) {
+		// Send the error to Sentry and then prevent the element from rendering
+		window?.guardian?.modules?.sentry?.reportError(error, 'onwards-lower');
+		return null;
+	}
+
+	if (loading) {
 		return (
-			<Container
-				heading={data.heading || data.displayname} // Sometimes the api returns heading as 'displayName'
-				trails={buildTrails(data.trails, limit)}
-				description={data.description}
-				ophanComponentName={ophanComponentName}
-				format={format}
-				isCuratedContent={isCuratedContent}
+			<Placeholder
+				// 300 is a best guess
+				height={300}
```

Comment:
Looking at a few pages, we tend to be closer to `600`.

Even with a maximum of three lines for cards in on “More on this story” and four for “More from Headlines”, you get to `540`.

Reply:
There are two sets of onwards cards so it's 300 + 300